lib/ansible/modules/cloud/google/gcp_compute_snapshot.py

```diff
@@ -167,9 +167,9 @@
 notes:
 - 'API Reference: U(https://cloud.google.com/compute/docs/reference/rest/v1/snapshots)'
 - 'Official Documentation: U(https://cloud.google.com/compute/docs/disks/create-snapshots)'
-- for authentication, you can set service_account_file using the c(gcp_service_account_file)
+- for authentication, you can set service_account_file using the C(gcp_service_account_file)
```

Comment:
`gcp_*` modules are auto-generated and these changes will be clobbered when this file is regenerated. You might want to fix these changes [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/2aea12555416b1d4300e95c4085cec432997a41c/provider/ansible/documentation.rb#L171)

Reply:
Good point, will fix it!

Reply:
Fixed, see [https://github.com/GoogleCloudPlatform/magic-modules/pull/2490](here)